docs/reference/indices/index-templates.asciidoc

```diff
@@ -1,18 +1,12 @@
 [[indices-templates]]
-=== Index templates
+= Index templates
```

Comment:
I think there's still room for improvement on this page. Few items:

* We talk about index settings, mappings, and index aliases. However, those topics haven't been explored in any depth earlier in the TOC. Maybe adding some xrefs there would help as a stopgap?

* An illustration or diagram may help explain the nested nature of index templates and component templates.

* We'll want to add a tutorial with corresponding Kibana UI screenshots to supplement the current snippet.